### PR TITLE
add jupyter notebook with code and instructions for creating and using data connections

### PR DESCRIPTION
**Pull request by gjbianco**

### CODE REVIEW

File: intro/projects-data/projects-data-nb.ipynb [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9c401286-c52e-42b0-9a51-505f2dd9cd15",
+   "metadata": {},
+   "source": [
+    "3. View a retrieved value.\n",
+    "Note that the `key_id` matches the value provided in the form when creating the data connection.\n",
+    "\n",
+    "> WARNING: Because cell outputs are saved as part of the notebook file, be cautious when printing sensitive information to notebook output.\n",
```
Comment (jramcast) [inline intro/projects-data/projects-data-nb.ipynb:70]: Good tip! We might want to include this in the lecture as well.

File: intro/projects-data/projects-data-nb.ipynb [hunk 2]
```diff
@@ -0,0 +1,248 @@
...
+   "id": "c1c591fb-f880-401c-b52f-e719ae83c753",
+   "metadata": {},
+   "source": [
+    "8. The pane to the left displays a new file called `new_hello.txt`.\n",
+    "Open the file and verify that its contents are `hello world`.\n",
+    "\n",
+    "> NOTE: You might need to refresh the file browser by clicking the `Refresh the file browser` button in the file browser pane.\n",
+    "The button displays as a circular arrow."
+   ]
+  },
```
Comment (jramcast) [inline intro/projects-data/projects-data-nb.ipynb:217]: Can we add a final markdown cell saying something like this?

> Return to the course book and finish the exercise.

I made the same suggestion in Alejandro's PR cause I forgot to go back and run the lab finish script.

https://github.com/RedHatTraining/AI26X-apps/pull/4/files#diff-873c83ef199f002867685dd407805c4428c2b39444e440e6b90b56fc982531d0R286

File: intro/projects-data/projects-data-nb.ipynb [hunk 3]
```diff
@@ -0,0 +1,248 @@
...
+    "for obj in objects[\"Contents\"]:\n",
+    "    print(obj[\"Key\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "63f252dd-3abe-4dd9-ad09-282047b6d97a",
+   "metadata": {},
+   "source": [
+    "> NOTE: Optionally, check the corresponding S3 bucket for the new `hello.txt` object."
```
Comment (jramcast) [inline intro/projects-data/projects-data-nb.ipynb:185]: Style

```suggestion
    "> NOTE: Optionally, verify the corresponding S3 bucket for the new `hello.txt` object."
```

**Review by jramcast — APPROVED** [review]
Good one Guy! I am approving upfront and leaving a couple of minor suggestions.

File: intro/projects-data/minio.yaml [hunk 4]
```diff
@@ -0,0 +1,161 @@
...
+  to:
+    kind: Service
+    name: minio-service
+    weight: 100
+  port:
+    targetPort: ui
+  wildcardPolicy: None
+  tls:
+    termination: edge
+    insecureEdgeTerminationPolicy: Redirect
```
Comment (jramcast) [inline intro/projects-data/minio.yaml:161]: I am curious, is this how you managed to solve the cert problem?

**Review by sffrench — APPROVED** [review]
LGTM

**Pull request merged**